### PR TITLE
931 aileron pos averaged

### PR DESCRIPTION
**Pull request by dany93**
Deleting the  'aileron pos averaged' branch (to modify the branch name) closed the #933 pull request.

### CODE REVIEW

File: c172p.xml [hunk 1]
```diff
@@ -1099,6 +1099,17 @@
             </product>
         </function>
 
+        <function name="fcs/aileron-pos-rad-avgd">
+            <description>Averaged aileron position</description>
+            <avg>
+                <property>fcs/left-aileron-pos-rad</property>
+                <product>
+                    <property>fcs/right-aileron-pos-rad</property>
```
Comment (onox) [inline c172p.xml]: Did you try `<property>-fcs/right-aileron-pos-rad</property>` (without any spaces between - and the property)?

Comment (dany93) [reply c172p.xml]: Yes.
It gives 

> Fatal exception, '-' found in propertyname after 'jsbsim' name must begin with alpha Il or '_'

Comment (onox) [reply c172p.xml]: It seems the minus sign is only supported for `<input>` in the summer, pure gain, etc. components. I'll merge the PR.

Comment (dany93) [reply c172p.xml]: I seemed to remember having seen this somewhere, but I couldn't find where.

**Comment by dany93** [comment]
Thank you, @onox .

**Pull request merged**